py/emitglue.h

```diff
@@ -85,6 +85,8 @@ typedef struct _mp_raw_code_t {
     uint16_t n_raw_code;
     #if MICROPY_PY_SYS_TRACE
     mp_bytecode_prelude_t prelude;
+    // Python source line where the raw_code was created e.g. MP_BC_MAKE_FUNCTION
+    // Required by the `call` trace event.
```

Comment:
"... This is different from lineno info stored in prelude, which provides line number for first statement of a function. Required to properly implement "call" trace event."

(Adjust to provide actual info, the point that it should explain why seemingly duplicate data is stored.)